src/pkg/user/service/service.go

```diff
@@ -22,77 +28,247 @@ type Service struct {
 	Proc  *processing.Proc
 }
 
+type TokenMetadata struct {
+	UUID   string
+	UserID string
+}
+
+type TokenDetails struct {
+	AccessToken  string
+	RefreshToken string
+	AccessUUID   string
+	RefreshUUID  string
+	AtExpires    int64
+	RtExpires    int64
+}
+
 const (
-	N            = 1048576
-	r            = 8
-	p            = 1
-	keyLen       = 32
-	saltLenBytes = 16
+	N      = 1048576
+	r      = 8
+	p      = 1
+	keyLen = 32
 )
 
 func NewUserService(cfg *config.Config, infra *infrastructure.Infra) *Service {
-	proc := processing.New(
+	process := processing.New(
```

Comment:
Why is it renamed? 
Proc is short for processing.
Process is a different meaning

Reply:
![image](https://user-images.githubusercontent.com/4029273/195855199-cdec6870-3d6b-49f0-82e6-27f0dfa9d072.png) got warning from IDE.